TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <Import Project="$(MSBuildExtensionsPath)\Microsoft.NuGet.props" />
+  
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x64</Platform>
+    <ProjectGuid>{D47E204B-D06C-4BEA-A6E0-1A1CFBFB193B}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>MSBuildTestApp</RootNamespace>
+    <AssemblyName>$(MSBuildThisFileName)</AssemblyName>
+    <TargetFrameworkIdentifier>NETCoreApp</TargetFrameworkIdentifier>
+    <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
+    <OutputPath>bin\Debug\netcoreapp1.0</OutputPath>
```

Comment:
This should be dependent on the Configuration.  We don't want Release builds going to 'bin\Debug\netcoreapp1.0`.

Although it would be nice to not have to specify this at all.